lib/drupalvm/vagrant.rb

```diff
@@ -95,7 +95,8 @@ def get_vhost_aliases(vconfig)
   if vconfig['drupalvm_webserver'] == 'apache'
     aliases = get_apache_vhosts(vconfig['apache_vhosts'])
   else
-    aliases = get_nginx_vhosts(vconfig['nginx_hosts'])
+    # @todo shim for `nginx_hosts`.
```

Comment:
The `@todo` here - is that just meant to say 'remove the shim part at some point in the future'?